kong/router/atc.lua

```diff
@@ -63,7 +63,7 @@ do
 
     ["String"] = {"net.protocol", "tls.sni",
                   "http.method", "http.host",
-                  "http.path", "http.raw_path",
```

Comment:
Not only this change, but when `execute` encounter a field it does not recognize, an error should be thrown (I think `assert` is appropriate here).

Reply:
Got, I will add check code in `select()`.

Reply:
Add a last `else` for error handling in `select()`.